### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by SteveChurch** [digitalocean/supabase-on-do#34] [General]
Not an issue! But a big thank you, I was just in the process of deploying Supabase within AWS using CDK but I would much prefer to place it with my other DO resources and this does the job nicely! 

🙇🏻

**Comment by timothymamo** [comment]
This made my day @SteveChurch, thanks. If I could pin it I would 😄.

**Comment by SteveChurch** [comment]
> This made my day @SteveChurch, thanks. If I could pin it I would 😄.

You're more than welcome! 😄
